package.json

```diff
@@ -85,6 +86,8 @@
     "typescript": "^3.8.3",
     "unist-util-visit": "^2.0.3",
     "unist-util-visit-parents": "^3.1.1",
+    "xmldom": "^0.6.0",
+    "xpath": "^0.0.32",
```

Comment:
I don't love that `xpath` feels a bit like abandonware,but this seems like the best option available.